lib/config.js

```diff
@@ -139,6 +139,9 @@ const JAMBONES_USE_FREESWITCH_TIMER_FD = process.env.JAMBONES_USE_FREESWITCH_TIM
 const JAMBONES_DIAL_SBC_FOR_REGISTERED_USER = process.env.JAMBONES_DIAL_SBC_FOR_REGISTERED_USER || false;
 const JAMBONES_MEDIA_TIMEOUT_MS = process.env.JAMBONES_MEDIA_TIMEOUT_MS || 0;
 const JAMBONES_MEDIA_HOLD_TIMEOUT_MS = process.env.JAMBONES_MEDIA_HOLD_TIMEOUT_MS || 0;
+// jambonz
+const JAMBONES_DIAL_TRANSCRIBE_EP_DESTROY_DELAY_MS =
+  process.env.JAMBONES_DIAL_TRANSCRIBE_EP_DESTROY_DELAY_MS || 1000;
```

Comment:
I would rather not have a default, since this changes existing behavior

Reply:
fixed